tests/mineral_test.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+from pycoal import mineral
+
+def test_normalize():
+    pass
+
+def test_indexOfGreaterThan():
```

Comment:
Can you make best efforts to clearly document all tests... it will make much more sense to everyone that way. Thank you @heidiaclayton